src/components/SubPage/EditingDesignSubpage.tsx

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import { Card, DescriptionsProps } from "antd";
+import { Diagram } from "../../types";
+import DiagramCard from "../shared/DiagramCard";
+import InfoPanel from "../shared/InfoPanel";
+
+const {
+    container,
+    card,
+    legendText,
+} = require("../../style/disease-subpage.module.css");
+
+const { pamSite, mutation } = require("../../style/editing-design.module.css");
+
+interface EditingDesignSubpageProps {
+    editingDesignData: {
+        crna_target_site: string;
+        dna_donor_sequence: string;
+        cas9: string;
+        f_primer: string;
+        r_primer: string;
+        diagrams: Diagram[];
```

Comment:
I think these should just be top level props. I also think the "convert-data.ts" could be a good place to take the database data and convert it into front end consumable data. It's where I've converted everything from snake case to camel case too